client/js/app/main/modules/createProject/createProject.service.js

```diff
@@ -0,0 +1,8 @@
+module.exports = function($http){
+
+    this.sendData = function (project) {
```

Comment:
make name of this method and file more clear, sendData in createService does not explain anything